clmm/theory/parent_class.py

```diff
@@ -425,7 +425,7 @@ def _eval_convergence(self, r_proj, z_cl, z_src):
         sigma_c = self.eval_critical_surface_density(z_cl, z_src)
         return sigma/sigma_c
 
-    def eval_reduced_tangential_shear(self, r_proj, z_cl, z_src, z_src_model='single_plane',
+    def eval_reduced_tangential_shear(self, r_proj, z_cl, z_src=None, z_src_model='single_plane',
```

Comment:
@combet If I make the same change at `compute_reduced_tangential_shear` in `func_layer.py`, I will have to change the order of the parameters, since the function is
````
compute_reduced_tangential_shear(
        r_proj, mdelta, cdelta, z_cluster, z_src, cosmo,
        delta_mdef=200, halo_profile_model='nfw', massdef='mean',
        z_src_model='single_plane', beta_s_mean=None, beta_s_square_mean=None,
        z_distrib_func=None, validate_input=True):
````

I will have to change such as the new order  will be `cosmo, z_src=None`, and thus we must change the examples in the notebook where the functions are called with a parameter order. Should I change the function and check all the notebooks?

Reply:
Hi @eduardojsbarroso - I think this is OK to change but would like @m-aguena and @marina-ricci's input as this means the API will not be backward compatible. Users currently using `compute_reduced_tangential_shear` will have to change the argument ordering for it to work. I don't think this is a major issue at this stage if properly tagged, but would be best to have agreement on this.

Reply:
Comment : We will implement that change in a separate PR.